react-frontend/src/components/Nav/covidbanner.js

```diff
@@ -1,81 +1,65 @@
+
+  const closeBanner = (event) => {
+    event.preventDefault();
+    setBreadCrumb(true);
+    setShowBanner(false);
+    const nav = document.getElementById('navbar');
+    nav.style.top = '67px';
+    const header = document.getElementById('navheader');
+    header.style.top = 0;
+    props.test(breadCrumb);
+  };
+
   if (showBanner) {
     return (
-      <CovidBannerStyle>
-        <div>
-          <Alert
-            style={{
-              backgroundColor: '#355992',
-              fontSize: '18px',
-              fontColor: 'white',
-              textAlign: 'center',
-            }}
-            message={
-              <CovidTextStyle>
-                <table align="center">
-                  <tr>
-                    <td width="8%" valign="top" align="right">
-                      <InfoCircleFilled />
-                    </td>
-                    <td width="80%">
-                      {' '}
-                      B.C. has declared a state of emergency. Learn about{' '}
-                      <CovidLinkStyle href="http://covid-19.bccdc.ca/?utm_campaign=20200323_GCPE_AM_COVID_7_NOTIFICATION_WORDPRESS_BCGOV_EN_BC__NOTIFICATION">
-                        COVID-19 health issues.
-                      </CovidLinkStyle>{' '}
-                      |{' '}
-                      <CovidLinkStyle href="https://www2.gov.bc.ca/gov/content/safety/emergency-preparedness-response-recovery/covid-19-provincial-support?utm_campaign=20200323_GCPE_AM_COVID_8_NOTIFICATION_WORDPRESS_BCGOV_EN_BC__NOTIFICATION">
-                        B.C.'s response to COVID-19.{' '}
-                      </CovidLinkStyle>
-                    </td>
-                    <td width="10%" align="center" valign="top">
-                      <CovidButtonStyle>
-                        <a
-                          href=""
-                          onClick={(event) => {
-                            event.preventDefault();
-                            setBreadCrumb(true);
-                            setShowBanner(false);
-                            const nav = document.getElementById('navbar');
-                            nav.style.top = '67px';
-                            const header = document.getElementById('navheader');
-                            header.style.top = 0;
-                            props.test(breadCrumb);
-                          }}
-                        >
-                          <font color="white">X</font>
-                        </a>
-                      </CovidButtonStyle>
-                    </td>
-                  </tr>
-                </table>
-              </CovidTextStyle>
-            }
-          />
-        </div>
-      </CovidBannerStyle>
+      <CovidAlert
+        message={
+          <Grid>
+            <Row>
+              <CovidCol xs={1}>
```

Comment:
![gap](https://user-images.githubusercontent.com/16946297/98304110-2dccdc80-1f74-11eb-97f9-38c67366ae49.png)

Sorry, this is maybe a little nit-picky, but I think the alignment might better match gov.bc.ca if you put `InfoCircleFilled` within the same div as the text and links? Like this:

![columns](https://user-images.githubusercontent.com/16946297/98304020-05dd7900-1f74-11eb-9a11-365031b22e22.png)

Reply:
![image](https://user-images.githubusercontent.com/9705602/98305952-a6816800-1f77-11eb-99f3-4ca9bc07ebbb.png)
I floated the icon right.  Instead to avoid more alignment issues.

Reply:
This works well. Would you mind also adding `fontSize: '1.5em'` to increase the size and improve the alignment of the icon?